Makefile

```diff
@@ -27,6 +27,10 @@ local:
 	mkdir -p bin
 	go build -ldflags "-X main.vendorVersion=${VERSION}" -o bin/gcfs-csi-driver ./cmd/
 
+windows-local:
+	mkdir -p bin
+	GOOS=windows GOARCH=amd64 go build -ldflags "-X main.vendorVersion=${VERSION}" -o bin/gcfs-csi-driver.exe ./cmd/
+
 push:
 	docker push $(IMAGE):$(VERSION)
```

Comment:
We will need to write a separate Dockerfile to build a Windows container. I haven't written one because the driver can't run from within a container on Windows. Can write one and include it if you'd like.

Reply:
It may be good to add now so once CSI on windows is ready, we can easily leverage this

Reply:
Added a Dockerfile for windows under test/experimental but the driver has to be compiled first. I added a target in the Makefile to do this.